mica/web/templates/mica/telem_table.html

```diff
@@ -0,0 +1,159 @@
+{% extends "base.html" %}
+
+{% block extra_css %}
+<style>
+table.pcad td {
+line-height: 1rem;
+padding: .4rem;
+font-size: .7rem;
+}
+table.pcad tr {
+background-color: transparent;
+}
+table.pcad {border-spacing: 5px;}
+</style>
+{% endblock %}
+
+{% block body %}
+<body class="with-background">
+{% endblock %}
+
+{% block content %}
+<p><p>
+<h1 align="center"> PCAD Telem Table </h1>
+
+<form action="" method="get">
+      <div class="row">
+        <div class="large-2 columns"> &nbsp; </div>
+        <div class="large-3 columns">
+          <label for="right-label1" class="right inline">Start Time</label>
+          <label for="right-label2" class="right inline">Stop Time</label>
+        </div>
+        <div class="large-3 columns">
+          <input type="text" id="right-label1" name="start_time"
+          value="{{start_time_date}}">
+          <input type="text" id="right-label2" name="stop_time" value="{{stop_time_date}}">
+        </div>
+        <div class="large-4 columns"> &nbsp; </div>
+      </div>
+      <input type="submit" value="Submit"  style="display: none;">
+</form>
+
+{% if events %}
+<TABLE class='pcad'>
+{% for event in events %}
+<TR><TD>{{ event }}</TD></TR>
```

Comment:
This could use clever formatting to get clean descriptions and links back to event details, but I didn't see already-available code to do that for any kind of event.